views/html/login.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Login</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="../vendor/bootstrap-4.3.1-dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="../css/style.css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
+        <script src="../vendor/jquery-3.3.1.min.js"></script>
+        <script src="../vendor/popper.min.js"></script>
+        <script src="../vendor/bootstrap-4.3.1-dist/js/bootstrap.min.js"></script>
+        <script src="../js/sample.js"></script>
+        <script src="https://apis.google.com/js/api:client.js"></script>
+    </head>
+    <body>
+        <br>
+        <div class="container">
+        <br>
+        <ul class="nav nav-pills mb-3"  id="pills-tab" role="tablist">
+            <li class="nav-item"><a class="nav-link active" data-toggle="pill" role="tab" href="#citizen" aria-controls="citizen" aria-selected="true"><h3>Citizen</h3></a></li>
+            <li class="nav-item"><a class="nav-link" data-toggle="pill" roll="tab" href="#organisation" aria-controls="organisation" aria-selected="false"><h3>Organisation</h3></a></li>
+        </ul>
+        <div class="tab content" id="pills-tabContent">
```

Comment:
the class name is `tab-content` there is no space but a hyphen. this will resolve your dynamic pill problem. Please check.